### PR TITLE
MISP-1132: Busterizing  QT5

### PR DESCRIPTION
**Pull request by robinmdh**
making qt5 work on debian buster for out R1 embeded platform.

### CODE REVIEW

**Review by raysiudak — DISMISSED** [review]
Documentation, thank you

File: build.sh [hunk 1]
```diff
@@ -87,10 +78,89 @@ build()
...
         -skip qtsystems \
-        -skip qtwebview
+        -platform linux-g++-64 \
+        -device ultimaker-linux-sun7i \
+        -device-option CROSS_COMPILE="${CROSS_COMPILE}" \
+        -sysroot "${SYSROOT}" \
+        -extprefix "${TARGET_DIR}/qt"
+
+#        -skip qtenginio \
+#        -skip qtquick1 \
```
Comment (CoenSchalkwijk) [inline build.sh:93]: Remove commented stuff?

File: build_sysroot.sh [hunk 2]
```diff
@@ -49,14 +49,16 @@ build_sysroot()
...
     cp -f "${QEMU_BINARY}" "${SYSROOT}/usr/bin"
 
     # Install the forked specially configured dependencies
     # TODO: These should also come from cloudsmith and have a correct version number
     cp "${TOOLS_DIR}/"*".deb" "${SYSROOT}"
 
-    chroot "${SYSROOT}" /usr/bin/dpkg -i /libdrm-ultimaker_2.4.102-imx6dl_armhf.deb
-    chroot "${SYSROOT}" /usr/bin/dpkg -i /mesa-ultimaker_19.0.1-imx6dl_armhf.deb
+#    chroot "${SYSROOT}" /usr/bin/dpkg -i /libdrm-ultimaker_2.4.102-imx6dl_armhf.deb
+#    chroot "${SYSROOT}" /usr/bin/dpkg -i /mesa-ultimaker_19.0.1-imx6dl_armhf.deb
```
Comment (CoenSchalkwijk) [inline build_sysroot.sh:61]:
```suggestion
```

**Review by CoenSchalkwijk — DISMISSED** [review]

File: build_sysroot.sh (hunk 2 above)
Comment (bryanfoley) [reply build_sysroot.sh:61]: Yeah, maybe I should move these to the README.md as a reminder of where to include the changes if you wish to build with these libs.

Comment (bryanfoley) [reply build_sysroot.sh]: Removed the commented code and moved the relevant information to the README in a new section at the bottom.

**Review by CoenSchalkwijk — APPROVED** [review]

**Review by raysiudak — APPROVED** [review]

**Pull request merged**